genesyscloud/team/data_source_genesyscloud_team.go

```diff
@@ -0,0 +1,41 @@
+package team
```

Comment:
Lets chat tomorrow because we need to also hook teams into the routing queue resource.  Routing queues allows you to assign teams, but we not currently support it

Reply:
deemed not needed